### DISCUSSION TITLE
Outbound IP address using IP Pools

### DISCUSSION THREAD

**Discussion by danielpotthast** [postalserver/postal#1622] [Help with using Postal] [answered]
Hello,

I've setup a virtual postal server (default server IPv4: `1.1.1.1`, IPv6: `1:1:1:1::1`) with two organizations.
Each organization should have its own IPv4 / IPv6 address for outbound email traffic.
They are already configured and assigned to the host system (Debian 10).

**Network interfaces:**
```
ip addr
1: lo: …
2: eth0: (…)
    inet 1.1.1.1/32 (…) eth0 (…)
    inet 2.2.2.2/32 (…) eth0:1 (…)
    inet 3.3.3.3/32 (…) eth0:2 (…)
    inet6 1:1:1:1::1/64 (…)
    inet6 2:2:2:2::1/64 (…)
    inet6 3:3:3:3::1/64 (…)
```

**IP Pools Configuration (`/ip_pools`):**
- Organization A: IPv4: 2.2.2.2, IPv6: 2:2:2:2::1, Hostname: postal.organization-a.com, Prio: 100
- Organization B: IPv4: 3.3.3.3, IPv6: 3:3:3:3::1, Hostname: postal.organization-b.com, Prio: 100

**Organization Configuration:**
- Organization IPs (`/org/organization-a/ip_pools`): IP Pool 'Organization A' is checked
- Organization IPs (`/org/organization-b/ip_pools`): IP Pool 'Organization B' is checked

When sending a test mail from 'Organization A Mail-Server' I would expect an email from 2.2.2.2 / 2:2:2:2::1, but it's delivered via 1:1:1:1::1.

Did I miss a configuration step?

Thank you! 🙏

**Comment by danielpotthast** [comment, marked as answer]
Okay, after some trial & error and with the help of discussion https://github.com/postalserver/postal/discussions/1444, I found the solution:

After setting up the IP Pools, you need to go to the server settings (`/org/organization-a/servers/test-mail-server/edit`) and simply click 'Save server' (without any changes). This saves the correct server-`ip_pool_id` to the MySQL database and fixes the outbound ip address issue.

**Reply by willpower232** [reply]
thanks for your research, interesting issue
